### PR TITLE
get user info from server

### PR DESCRIPTION
**Pull request by lbai001**

### CODE REVIEW

File: auth.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const request = require('request');
+const express = require('express');
+require('dotenv').config();
+
+module.exports = () => {
+  const router = new express.Router();
+  router.post('/getAuth0User', (req, res, next) => {
```
Comment (RobAWilkinson) [inline auth.js]: why a post to `get` the user?

File: authmiddleware.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      cache: true,
+      rateLimit: true,
+      jwksRequestsPerMinute: 5,
+      jwksUri: process.env.JWKS_URI
+    }),
+    aud: process.env.AUDIENCE,
+    issuer: process.env.ISSUER,
+    algorithms: ['RS256']
+  })
+};
```
Comment (RobAWilkinson) [inline authmiddleware.js]: does this just return a 401 if the jwt is invalid?

File: server.js [hunk 3]
```diff
@@ -8,7 +8,10 @@ let rpc = require('./rpc');
 const printer = require('./printer');
 const logger = require('./logger.js');
 const CONFIG = require('./config.js');
+const checkJWT = require('./authmiddleware.js').verify;
```
Comment (RobAWilkinson) [inline server.js]: lets load this in the controllers that need it.

**Review by RobAWilkinson — COMMENTED** [review]
Also, please rename authmiddleware to `authMiddlewares` since its set up to have more than one function in there.

File: authmiddleware.js (hunk 2 above)
Comment (lbai001) [reply authmiddleware.js]: 500 and a message

File: server.js [hunk 4]
```diff
@@ -9,8 +9,9 @@ const printer = require('./printer');
 const logger = require('./logger.js');
 const CONFIG = require('./config.js');
 const dbWrite = require('./dbWrite');
-const PORT = 3000;
+require('dotenv').config();
 
+const PORT = 3000;
```
Comment (RobAWilkinson) [inline server.js]: can you bring this into the env as well?

**Pull request merged**